### ISSUE TITLE
Animation doesn't work on Android device.

### ISSUE THREAD

**Issue by dg2k** [ewerspej/epj.ProgressBar.Maui#3]
I couldn't trace back to where the animation stopped working as I changed from net8.0 to net7.0 and did updates of several Maui packages.

The thing is that everything works as expected on Android Emulator but not Android device wrt animation. It's easy to think that it's the device at fault, but why and how?

Just wondering if there is something known about animations wrt devices. Any workaround to try out would be appreciated.

Visual Studio IDE:   Version 17.6.5 (Visual Studio 2022)
.NET Framework:    Version 4.8.09032
SDK Version:          net7.0 (7.0.400-preview.23330.10)
Android Device:     Samsung M325G, Android API 33

**Comment by ewerspej** [comment]
I am not aware of any such problem. It's always curious when things go well on the emulator but not on a real device. Are you running the emulator with the same Android version as the real device? I don't know what side effects downgrades from .NET 8.0 preview to .NET 7.0 can occur. Anyway, this is not a scenario that I can support, because .NET 8.0 is still in preview and to me it doesn't seem like a problem specific to the Progress Bar control.

Does this also happen when you create a fresh project using the Progress Bar using .NET 7.0?

**Comment by dg2k** [comment]
I found out the culprit to be the Battery Saver Mode for Android which I set on travel and forgot.

Did Google tell us that animation stops when battery saver is on? May be in some hidden documentation as their own AI assisted search turned up nothing.

Now I am learning the hard way and won't forget that. No wonder why the Emulator was working but not the physical Device. The physical device which I modified wrt animation.
